installers/toradex-apalis/install-avena

```diff
@@ -0,0 +1,16 @@
+# Reset and save the Das U-Boot environment
+env default -a
+saveenv
+
+# Try to automatically select the correct Apalis device tree
+if test $board_rev -ge "010a" -a $board_rev -le "010z"; then
+  setenv fdtfile imx6q-apalis-eval.dtb
+elif test $board_rev -ge "011a" -a $board_rev -le "011z"; then
+  setenv fdtfile imx6q-apalis-eval.dtb
```

Comment:
You are setting the device tree to the same file no matter the board revision?

Reply:
Great catch, thanks! Hopefully this fixes a bug about mmcblk1 being detected as mmcblk2 within the installer (and breaking writing to u-boot directly)